components/Hackathons.js

```diff
@@ -121,7 +155,7 @@ const MobileDashedConnector = styled.div`
 const HackathonImage = styled.img`
   width: 100%;
   height: 100%;
-  filter: blur(3px);
+  filter: blur(2.5px);
```

Comment:
Can you just confirm that the planets still have the right amount of blur ? Since I changed them from svg to png they may be rendering differently

Reply:
i think it looks good? i'll ask design!